foyer/forcefield.py

```diff
@@ -308,6 +308,8 @@ def apply(self, topology, references_file=None, *args, **kwargs):
         references_file : str, optional, default=None
             Name of file where force field references will be written (in Bibtex
             format)
+        use_residue_map : boolean
+            Bypass atomtyping duplicate residues using a residue map
```

Comment:
Could you maybe clarify a bit more what this does via the variable name and/or the description here? Maybe something along the lines of `copy_atomtypes_to_duplicate_residues`